### PR TITLE
Mojang.BedrockDedicatedServer version 1.19.81.01

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107115)

### CODE REVIEW

File: manifests/m/Mojang/BedrockDedicatedServer/1.19.81.01/Mojang.BedrockDedicatedServer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Mojang.BedrockDedicatedServer
+PackageVersion: 1.19.81.01
+PackageLocale: en-US
+Publisher: Mojang
+PackageName: Bedrock Dedicated Server
```
Comment (stephengillie) [inline manifests/m/Mojang/BedrockDedicatedServer/1.19.81.01/Mojang.BedrockDedicatedServer.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.minecraft.net/
PublisherSupportUrl: https://feedback.minecraft.net/
PackageName: Bedrock Dedicated Server
PackageUrl: https://www.minecraft.net/en-us/download/server/bedrock
```

**Comment by Trenly** [comment]
> image site is not down

They may block automated requests like the ones that come from the pipelines / winget

**Comment by stephengillie** [comment]
This package is a non-interactive game server:
```
NO LOG FILE! - setting up server logging...
[2023-05-18 09:06:59:149 INFO] Starting Server
[2023-05-18 09:06:59:150 INFO] Version 1.19.81.01
[2023-05-18 09:06:59:152 INFO] Session ID a4ec21ab-7ba5-460e-a985-414747011c00
[2023-05-18 09:06:59:155 INFO] Build ID 1.19.81
[2023-05-18 09:06:59:157 INFO] Branch r/19_u8
[2023-05-18 09:06:59:159 INFO] Commit ID 2606f6a1a1e3ecf907cb776d138dac765c5ba3ff
[2023-05-18 09:06:59:162 INFO] Configuration Publish
[2023-05-18 09:06:59:166 INFO] Level Name: Bedrock level
[2023-05-18 09:06:59:170 INFO] Game mode: 0 Survival
[2023-05-18 09:06:59:172 INFO] Difficulty: 1 EASY
...
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**